### PR TITLE
Change SearchResult interface

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: changelog/unreleased/enhancement-search-all-files-announce-limit [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Enhancement: Search all files announce limit
+
+We announce to the user if the search results in the "search all files" search provider exceeds the limit of 200 items.
```
Comment (kulmann) [inline changelog/unreleased/enhancement-search-all-files-announce-limit:3]:
```suggestion
If the search results of a backend search exceed the limit of 200 items we now announce that additional items exist.
```

File: packages/web-app-search/src/types.ts [hunk 2]
```diff
@@ -1,18 +1,25 @@
-export interface SearchResult {
+export interface SearchResultValue {
   id: string
   data: unknown
 }
 
+export interface SearchResult {
+  meta?: {
```
Comment (kulmann) [inline packages/web-app-search/src/types.ts:7]: I thought that you and @fschade decided against a `meta` prop? 🤔 Is this PR up to date and ready for review?

Comment (JanAckermann) [reply packages/web-app-search/src/types.ts:7]: We had a call after that, and I show-case a proof of concept, @fschade was alright with that.

> Is this PR up to date and ready for review?
Yes

**Review by kulmann — APPROVED** [review]
👍

**Pull request merged**